fluentui_ccb/src/main/java/com/microsoft/fluentui/tokenized/contextualcommandbar/ContextualCommandBar.kt

```diff
@@ -0,0 +1,429 @@
+package com.microsoft.fluentui.tokenized.contextualcommandbar
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.interaction.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.onFocusEvent
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import com.microsoft.fluentui.icons.CCBIcons
+import com.microsoft.fluentui.icons.ccbicons.Keyboarddismiss
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.StateColor
+import com.microsoft.fluentui.theme.token.controlTokens.ContextualCommandBarTokens
+import kotlinx.coroutines.launch
+import kotlin.math.max
+
+val LocalContextualCommandBarTokens = compositionLocalOf { ContextualCommandBarTokens() }
+
+enum class ActionButtonState {
```

Comment:
ActionButtonPlacement sounds more logical in the CCB context. Please rename it

Reply:
This is not only about placement but also for no action button Case. Can you suggest a better alternative?

Reply:
"None" suggest that it is not placed for not having action button.
"Start" & "End" suggest where it is placed. 
Is this correct understanding?